teeracle/src/tests.rs

```diff
@@ -59,7 +74,8 @@ fn verifiy_update_exchange_rate_works() {
 fn verifiy_get_existing_exchange_rate_works() {
```

Comment:
I just noticed some typos in the verify. :)

Reply:
I am not fond of the verify prefix anyhow. I don't think it adds information, but only lengthens the function name unnecessarily.

Reply:
OK. I remove then all